Sources/AsyncQueue/ActorQueue.swift

```diff
@@ -0,0 +1,167 @@
+// MIT License
+//
+// Copyright (c) 2022 Dan Federman
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+/// A queue that executes asynchronous tasks enqueued from a nonisolated context.
```

Comment:
It would be nice I think for the doc comment to include sample code like in https://github.com/dfed/swift-async-queue/pull/3/files#r1033055297 to show how this class is meant to be used.

Reply:
I'll take a documentation pass in the next couple days. Agreed this documentation needs some strengthening.

Reply:
Welp I've been slow here. Going to do a documentation pass today.

Reply:
Took a pass here: 744db4a

I put the code in the README rather than the the doc comment since this is a relatively complex example for a doc comment. That said, if you've got examples of code that has built complex code samples into doc comments, please do link me!

There's still room for improvement in the docs, but I think we're in a place where we could land this and improve over time.

Reply:
`AsyncStream` is a GREAT example of putting this documentation on the type in the code. I'll give this another pass, though I think we can likely merge as-is and take another docs pass post-merge.

Reply:
Took another pass because why not: dca625d